packages/atomic/src/components/insight/atomic-insight-interface/e2e/page-object.ts

```diff
@@ -39,7 +41,9 @@ export class InsightInterfacePageObject extends BasePageObject<'atomic-insight-i
   }
 
   get insightResults() {
-    return this.page.locator('atomic-insight-result');
+    return this.page.locator(
+      'atomic-insight-result:not(atomic-insight-result atomic-insight-result)'
+    );
```

Comment:
This selector was also updated to not match any of the folded child results. Only parents by adding the `:not()` part.